### ISSUE TITLE
Multiple ignore in single settings section

### ISSUE THREAD

**Issue by agrawalarpit14** [coala/coala#5841]
```
[section]
ignore = aaa
ignore = bbb
```
Expected behavior:
It should throw an error

Current behavior:
Both aaa & bbb gets ignored.

**Comment by agrawalarpit14** [comment]
@jayvdb The approach to solve this issue will be very different if I take the changes done in #5815 into account and if I don't. I think going forward with the former makes more sense?
Further let me know when such a situation is encountered: 
```
[section]
ignore = aaa
ignore = bbb
```
Should it give an error or should it contain `bbb`.

**Comment by jayvdb** [comment]
@agrawalarpit14 , it should be an error, I believe (so far).  This issue isnt about `+=` .  it is about overwriting config

The following should be possible of course.
```
[section]
ignore = aaa

[section.override]
ignore = bbb
```

Let us know if any tests depend on the same setting being overwritten.  We may need to re-evaluate.

**Comment by jayvdb** [comment]
There is a test which deals with this; see https://github.com/coala/coala/pull/5811#pullrequestreview-169054261

**Comment by agrawalarpit14** [comment]
@jayvdb I don't think this is an issue presently. I am not able to reproduce it in any scenario.
Presently, it overwrites the value and throws the following warning.
```
                    logging.warning('{} setting has already been defined in '
                                    'section {}. The previous setting will be '
                                    'overridden.'.format(key,
                                                         current_section.name))
```
And, it works as expected, it keeps the value stated later and discards the earlier one. Do you want me to change it into an error, or is the warning fine?

**Comment by akshatkarani** [comment]
@agrawalarpit14  Still working on this? Otherwise I would like to take this up. ;)

**Comment by agrawalarpit14** [comment]
Sure @akshatkarani you can work on this. 🙂🙂
